UPGRADE-14.0.md

```diff
@@ -45,3 +45,6 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 -->
 
 ## [Upgrade from v13.0.0 to v14.0.0-dev](https://github.com/shopsys/shopsys/compare/v13.0.0...14.0)
+
+- added test to keep elasticsearch converter and mapping in sync ([#2880](https://github.com/shopsys/shopsys/pull/2880))
```

Comment:
Upgrade notes should be in style of command, so it should be something like "add test to keel Elasticsearch converter and mapping in sync"

Reply:
👍  changed, thanks!